runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -559,18 +559,143 @@ xlpSubOptionsParser(J9JavaVM *vm, IDATA xlpIndex, XlpError *xlpError, UDATA *req
 	return xlpErrorState;
 }
 
+/**
+ * Given the requested size of the large pages to be used, the method will verify that value is supported by the system. 
+ * If the value is not supported, the system will set preffered Large Page size if configured on the system.
+ * The user will also be notified of the altered value being used.
+ */
+void
+verifyLargePageSize(J9JavaVM *vm, UDATA requestedPageSize, UDATA requestedPageFlags, MM_GCExtensions *extensions)
```

Comment:
Please remove GC Extensions from parameters. Use this way to get extensions instead:
```
MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(vm);
```

Reply:
Done. See https://github.com/eclipse/openj9/pull/7476/commits/cad6beaec1cc182b1a5325f005ef62dba56d2810